spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkScan.java

```diff
@@ -49,9 +77,14 @@ public TestSparkScan(String format) {
     this.format = format;
   }
 
+  @Before
+  public void useCatalog() {
+    sql("USE %s", catalogName);
+  }
+
   @After
   public void removeTables() {
-    sql("DROP TABLE IF EXISTS %s", tableName);
+    sql("DROP TABLE IF EXISTS %s PURGE", tableName);
```

Comment:
Why was this needed? We discourage the use of PURGE in general. Are you reusing table names or locations?

Reply:
Just want to cleanup files as well. Reverted.